### ISSUE TITLE
When installed as system app, Downloads no longer seems to work

### ISSUE THREAD

**Issue by Eliminater74** [Grarak/KernelAdiutor#173]
I build roms, When I compiled this with Android Studio , I tested the apk out and it worked no problem.
but when I did a product_copy_files over to system with rom build..
everything worked.. but downloads doesnt show up anymore.. So I am thinking that
that when its system app it doesnt detect device or something and wont show the downloads.

**Comment by Grarak** [comment]
That's not an issue. Apk on play store is signed. Your source compiled apk is not. That's why it fails.

**Comment by Eliminater74** [comment]
It never failed till I made it a system app. signed or not signed, I had the downloads working up till the point of making it a system app.. I was only letting you know about that.. when the app is installed to /system/app/KernelAdiutor   it failed to work with downloads anymore.. Being signed or not signed  shouldnt make a differents on that part. I followed your instructions on the downloads and make the adjustments according to it. I had it working 100%, up till the installed to System/APP..
